tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27-ondemand, py27-subscription, py27-unittests, pep8
+envlist = docs, py27-ondemand, py27-subscription, py27-unittests, pep8
```

Comment:
The reason I'm adding `docs` at the start of the envlist is it generally runs very quickly. I'd perhaps move `pep8` up to the start too